### ISSUE TITLE
UnmetDependency - taskid:fasta-annotate:Rimi.v1.02.fasta.dammit.fasta[dammit.annotate:ERROR]

### ISSUE THREAD

**Issue by macmanes** [camillescott/dammit#44]
getting an error with dammit at the very end of the annotation process:

```
$ dammit annotate Rimi.v1.02.fasta --busco-group vertebrata --n_threads 60 --database-dir /mnt/data3/macmanes/dammit_databases/ --full

/usr/local/lib/python2.7/dist-packages/matplotlib/font_manager.py:273: UserWarning: Matplotlib is building the font cache using fc-list. This may take a moment.
  warnings.warn('Matplotlib is building the font cache using fc-list. This may take a moment.')
========================================
dammit! a tool for easy de novo transcriptome annotation
Camille Scott 2015
========================================

~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
submodule: annotate
~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~

--- Checking PATH for dependencies

          [x] TransDecoder

          [x] LAST

          [x] HMMER

          [x] Infernal

          [x] crb-blast

          [x] BLAST+

          [x] BUSCO


--- Dependency results

          All dependencies satisfied!


--- Checking for database prep (dir: /mnt/data3/macmanes/dammit_databases)

          [x] download_and_gunzip:Pfam-A.hmm

          [x] hmmpress:Pfam-A.hmm

          [x] download_and_gunzip:Rfam.cm

          [x] cmpress:Rfam.cm

          [x] download_and_gunzip:aa_seq_euk.fasta

          [x] lastdb:aa_seq_euk.fasta.db

          [x] download_and_gunzip:ODB8_EukOGs_genes_ALL_levels.txt

          [x] download_and_untar:dammit_databases-vertebrata

          [x] download_and_gunzip:uniref90.fasta

          [x] lastdb:uniref90.fasta.db

--- Database results

          All databases prepared!


--- Running annotate!

          Transcriptome file: Rimi.v1.02.fasta

          Output directory: /mnt/data3/macmanes/imitator/Jan16/Rimi.v1.02.fasta.dammit

          [ ] transcriptome_stats:Rimi.v1.02.fasta

          [ ] busco:Rimi.v1.02.fasta-vertebrata

          [ ] transeq:Rimi.v1.02.fasta

          [ ] TransDecoder.LongOrfs:Rimi.v1.02.fasta

          [ ] hmmscan:longest_orfs.pep.x.Pfam-A.hmm

          [ ] remap_hmmer:longest_orfs.pep.pfam.tbl

          [ ] TransDecoder.Predict:Rimi.v1.02.fasta

          [ ] cmscan:Rimi.v1.02.fasta.x.Rfam.cm

          [ ] lastal:Rimi.v1.02.fasta.x.orthodb.maf

          [ ] lastal:Rimi.v1.02.fasta.x.uniref.maf

          [ ] maf_best_hits:Rimi.v1.02.fasta.x.orthodb.maf-
          Rimi.v1.02.fasta.x.orthodb.maf.best.csv

          [ ] maf-gff3:Rimi.v1.02.fasta.x.orthodb.maf.gff3

          [ ] maf_best_hits:Rimi.v1.02.fasta.x.uniref.maf-
          Rimi.v1.02.fasta.x.uniref.maf.best.csv

          [ ] maf-gff3:Rimi.v1.02.fasta.x.uniref.maf.best.csv.gff3

          [ ] cmscan-gff3:Rimi.v1.02.fasta.rfam.tbl.gff3

Some tasks failed![dammit.annotate:ERROR]
TaskError - taskid:remap_hmmer:longest_orfs.pep.pfam.tbl[dammit.annotate:ERROR]
PythonAction Error
Traceback (most recent call last):
  File "/usr/local/lib/python2.7/dist-packages/doit/action.py", line 383, in execute
    returned_value = self.py_callable(*self.args, **kwargs)
  File "/usr/local/lib/python2.7/dist-packages/dammit/tasks.py", line 472, in cmd
    assert(all(hmmer_df.env_coord_to <= merged_df.end))
AssertionError
[dammit.annotate:ERROR]
Some tasks failed![dammit.annotate:ERROR]
UnmetDependency - taskid:hmmscan-gff3:Rimi.v1.02.fasta.pfam.csv.gff3[dammit.annotate:ERROR]
remap_hmmer:longest_orfs.pep.pfam.tbl
[dammit.annotate:ERROR]
Some tasks failed![dammit.annotate:ERROR]
UnmetDependency - taskid:gff3-merge:Rimi.v1.02.fasta.dammit.gff3[dammit.annotate:ERROR]
hmmscan-gff3:Rimi.v1.02.fasta.pfam.csv.gff3
[dammit.annotate:ERROR]
Some tasks failed![dammit.annotate:ERROR]
UnmetDependency - taskid:fasta-annotate:Rimi.v1.02.fasta.dammit.fasta[dammit.annotate:ERROR]
gff3-merge:Rimi.v1.02.fasta.dammit.gff3
[dammit.annotate:ERROR]
```

**Comment by camillescott** [comment]
Thanks @macmanes -- looks like I made a mistake in my coordinate translation code, should be an easy fix.

**Comment by macmanes** [comment]
Thanks @camillescott. Look forward to the fix.

**Comment by macmanes** [comment]
did this ever get fixed?

**Comment by camillescott** [comment]
I just pushed the fix to PyPI in v0.2.6; turns out it was sitting on my work computer mostly done, and I hadn't pushed it off yet (it's been a busy semester...). Let me know how it goes!
